src/main/resources/templates/hint/list.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="cz"
+      xmlns:th="http://www.springframework.org/schema/data/jaxb">
+<head>
+    <meta charset="UTF-8">
+    <title>HintList</title>
+</head>
+<body>
+<h2>Šifra <span th:text="${cypher.name}">cypherName</span></h2>
```

Comment:
cypherName tam asi taky nema byt ne?

Reply:
viz výše, je to jen pro kontrolu vzhledu behem ladeni stranky